### PR TITLE
Merge remote-tracking branch 'origin/1.3' into 1.x

### LINKED ISSUE #1520: [BUG] 1.x should contain changes from 1.3
**What is the bug?**
From the [project branching guidance](https://github.com/opensearch-project/.github/blob/main/RELEASING.md#branching), security-dashboard-plugin's 1.3 branch would have changes propagated from 1.x.  Looking at the current state of the branches many changes have been pushed directly from main into 1.3

**How can one reproduce the bug?**
1. Inspect a comparison of the branches https://github.com/opensearch-project/security-dashboards-plugin/compare/1.x...1.3
1. There should be trivial differences between the branches, whereas there are ~30 commits difference

**What is the expected behavior?**
All changes in 1.3 should also be present in 1.x

**Do you have any additional context?**
- Coming from https://github.com/opensearch-project/security-dashboards-plugin/pull/1514

### PR DESCRIPTION
**Pull request by peternied**
### Description
Makes sure all changes that have been merged into 1.3 are also present in 1.x.  Note; this change should be manually merged, **not squashed**.

### Issues Resolved
- Resolves #1520 

### Check List
- [ ] ~New functionality includes testing~
- [ ] ~New functionality has been documented~
- [X] Commits are signed per the DCO using --signoff

### CODE REVIEW

**Comment by peternied** [comment]
Marking as draft to prevent accidental squash merge

File: package.json [hunk 1]
```diff
@@ -1,10 +1,10 @@
 {
   "name": "opensearch-security-dashboards",
-  "version": "1.3.0.0",
+  "version": "1.3.11.0",
```
Comment (cwperks) [inline package.json:3]: Should the version on 1.x be referring to 1.4?

**Review by cwperks — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (peternied) [reply package.json:3]: Unclear, but I'd say out of scope to get the 1.x branch unblocked.

**Comment by peternied** [comment]
### Manually pushed merge commit into 1.x directly

```
% git push origin HEAD:1.x
Enumerating objects: 1, done.
Counting objects: 100% (1/1), done.
Writing objects: 100% (1/1), 278 bytes | 278.00 KiB/s, done.
Total 1 (delta 0), reused 0 (delta 0), pack-reused 0
remote: Bypassed rule violations for refs/heads/1.x:
remote: 
remote: - Changes must be made through a pull request.
remote: 
To github.com:opensearch-project/security-dashboards-plugin.git
   80a273e..cbe06da  HEAD -> 1.x
```

**Pull request merged**